### PR TITLE
Add Title Banner and Museum layout to other pages

### PR DESCRIPTION
**Pull request by taesungh**
Continuing from #70,

Add Title Banner to other pages
- Unify museum container styles for body content

Refactor main layout elements with MuseumRoom
- Create `MainLayout` to include Navigation, Museum Room, and Footer
- Rotate Carousel Gallery for Home page

Fix main content height
 - Use flex display to ensure main body grows for Footer to be at bottom
 - Adjust MuseumRoom height to be one-sided extension for ceiling drop

### CODE REVIEW

File: src/styles/hackuci.scss [hunk 1]
```diff
@@ -30,3 +30,14 @@ h2,
 		box-shadow: 0px 8px 17px rgba(0, 0, 0, 0.37);
 	}
 }
+
+.museum-container {
+	max-width: 720px;
+	@include bootstrap.padding(3rem);
```
Comment (samderanova) [inline src/styles/hackuci.scss:36]: Question: What's the difference between this and `padding: 3rem;`?

**Review by samderanova — APPROVED** [review]

File: src/views/apply/Apply.tsx [hunk 2]
```diff
@@ -28,22 +27,27 @@ function Apply() {
...
-					<ApplicationPreface
-						isLoggedIn={isLoggedIn}
-						setAcceptedPreface={setAcceptedPreface}
-					/>
-				) : (
-					<ApplicationForm setSubmittedApplication={setSubmittedApplication} />
-				)}
+			<TitleBanner>
+				<h1>Apply</h1>
+			</TitleBanner>
```
Comment (samderanova) [inline src/views/apply/Apply.tsx:46]: Might just be me, but the light is somewhat off here compared to the one on the home page:
image
vs.
image

**Review by samderanova — CHANGES_REQUESTED** [review]

File: src/styles/hackuci.scss (hunk 1 above)
Comment (taesungh) [reply src/styles/hackuci.scss:36]: The mixin is a shorthand for [Bootstrap's RFS system](https://getbootstrap.com/docs/5.3/getting-started/rfs/) where the amount is calculated based on viewport width.

File: src/views/apply/Apply.tsx (hunk 2 above)
Comment (taesungh) [reply src/views/apply/Apply.tsx:46]: Maybe someone else could have a shot at this, but I'm not bothered enough to fix this right now 😣.

File: src/components/TitleBanner/TitleBanner.module.scss [hunk 3]
```diff
@@ -1,16 +1,28 @@
...
+	margin-bottom: calc(4rem + 4.25vw - 21vh);
+	margin-top: 8vh;
+	margin-left: auto;
+	margin-right: auto;
+
 	.lamp {
 		position: absolute;
 		margin: auto;
-		top: -8%;
+		top: -10%;
```
Comment (rileysw) [inline src/components/TitleBanner/TitleBanner.module.scss:22]: Not exactly an ideal solution, but to address Sam's comment, we can change this to a non-percentage value so at least it seems attached to the ceiling 🤷 (I tried with -6vh)

**Review by rileysw — APPROVED** [review]

**Review by samderanova — APPROVED** [review]

File: src/components/TitleBanner/TitleBanner.module.scss (hunk 3 above)
Comment (taesungh) [reply src/components/TitleBanner/TitleBanner.module.scss:22]: Thanks for the thought, maybe something we could fix later.

**Pull request merged**